stdlib/public/core/StringCreate.swift

```diff
@@ -120,8 +120,15 @@ extension String {
       )
       return result.asString
     case .error(let initialRange):
-      //This could be optimized to use excess tail capacity
-      return repairUTF8(result.codeUnits, firstKnownBrokenRange: initialRange)
+      // repairUTF8 will call reserveCapacity, invalidating result.codeUnits if
+      // we use it directly. This could be a little faster, but this error path
+      // shouldn't be perf-sensitive anyway.
+      let temporaryCodeUnits = Array(result.codeUnits)
+      defer { _fixLifetime(temporaryCodeUnits) }
```

Comment:
Why do we need _fixLifetime here?

Reply:
Ah, right you are, it does that internally